app/services/login/after_login_actions.rb

```diff
@@ -28,6 +28,12 @@ def perform
         TestUserDashboard::UpdateUser.new(current_user).call(Time.current)
         TestUserDashboard::AccountMetrics.new(current_user).checkout
       end
+
+      if Flipper.enabled?(:mhv_medical_records_phr_refresh_on_login,
+                          @current_user) && current_user.mhv_correlation_id.present?
+        MHV::PhrUpdateJob.perform_async(current_user.icn,
+                                        current_user.mhv_correlation_id)
```

Comment:
I don't think you need to gate this on `mhv_correlation_id` if you're also checking it during the job itself